<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterDlg</class>
 <widget class="QDialog" name="ParameterDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>455</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Project Settings</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>247</y>
     <width>541</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>70</y>
      <width>428</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Universal Primer:</string>
       </property>
       <property name="buddy">
        <cstring>uniprimerLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="uniprimerLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Barcode Length:</string>
       </property>
       <property name="buddy">
        <cstring>barcodelenSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="barcodelenSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>16</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>100</y>
      <width>519</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Min Read Depth:</string>
       </property>
       <property name="buddy">
        <cstring>mindepthSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="mindepthSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Max Read Depth:</string>
       </property>
       <property name="buddy">
        <cstring>maxdepthSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="maxdepthSpinBox">
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Flanking Length:</string>
       </property>
       <property name="buddy">
        <cstring>flankinglengthSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="flankinglengthSpinBox">
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>40</y>
      <width>483</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="muscleLabel">
       <property name="text">
        <string>MUSCLE:</string>
       </property>
       <property name="buddy">
        <cstring>muscleLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="muscleLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="muscleButton">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Padding Sequence:</string>
       </property>
       <property name="buddy">
        <cstring>paddingEditLine</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="paddingEditLine">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>130</y>
      <width>445</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Match Score:</string>
       </property>
       <property name="buddy">
        <cstring>matchscoreSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="matchscoreSpinBox">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Mismatch Score:</string>
       </property>
       <property name="buddy">
        <cstring>mismatchscireSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="mismatchscireSpinBox">
       <property name="minimum">
        <number>-5</number>
       </property>
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="value">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Gap Score:</string>
       </property>
       <property name="buddy">
        <cstring>gapscoreSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="gapscoreSpinBox">
       <property name="minimum">
        <number>-5</number>
       </property>
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="value">
        <number>-1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>10</y>
      <width>517</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="seqtypeLable">
       <property name="text">
        <string>File Format:</string>
       </property>
       <property name="buddy">
        <cstring>fastqRadioBtn</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="fastqRadioBtn">
       <property name="text">
        <string>FASTQ</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">filetypeButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="fastaRadioBtn">
       <property name="text">
        <string>FASTA</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">filetypeButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Score Type:</string>
       </property>
       <property name="buddy">
        <cstring>phred33RadioBtn</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="phred33RadioBtn">
       <property name="text">
        <string>phred33</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">scoretypeGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="phred64RadioBtn">
       <property name="text">
        <string>phred64</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">scoretypeGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>160</y>
      <width>281</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Max Mismatch:</string>
       </property>
       <property name="buddy">
        <cstring>maxmismatchSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="maxmismatchSpinBox">
       <property name="maximum">
        <number>5</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Threads</string>
       </property>
       <property name="buddy">
        <cstring>threadSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="threadSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="okayButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="moreButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>90</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Advanced &gt;&gt;</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>403</x>
     <y>10</y>
     <width>20</width>
     <height>221</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>44</y>
     <width>383</width>
     <height>95</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="inputfileLabel">
        <property name="text">
         <string>Sequencing Files:</string>
        </property>
        <property name="buddy">
         <cstring>addseqfileButton</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addseqfileButton">
        <property name="text">
         <string>Add File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeseqfileButton">
        <property name="text">
         <string>Remove File</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="seqfileListWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>144</y>
     <width>381</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="barcodefileLabel">
      <property name="text">
       <string>Barcode File:</string>
      </property>
      <property name="buddy">
       <cstring>barcodefileLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="barcodefileLineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="barcodefileButton">
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>204</y>
     <width>381</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="outfolderLabel">
      <property name="text">
       <string>Output Folder:</string>
      </property>
      <property name="buddy">
       <cstring>outfolderLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="outfolderLineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="outfolderButton">
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>174</y>
     <width>381</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="primerfileLabel">
      <property name="text">
       <string>Primer File:</string>
      </property>
      <property name="buddy">
       <cstring>primerfileLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="primerfileLineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="primerfileButton">
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>12</y>
     <width>240</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Project Name:</string>
      </property>
      <property name="buddy">
       <cstring>projnameLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="projnameLineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>projnameLineEdit</tabstop>
  <tabstop>addseqfileButton</tabstop>
  <tabstop>removeseqfileButton</tabstop>
  <tabstop>barcodefileButton</tabstop>
  <tabstop>primerfileButton</tabstop>
  <tabstop>outfolderButton</tabstop>
  <tabstop>okayButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>moreButton</tabstop>
  <tabstop>fastqRadioBtn</tabstop>
  <tabstop>fastaRadioBtn</tabstop>
  <tabstop>phred33RadioBtn</tabstop>
  <tabstop>phred64RadioBtn</tabstop>
  <tabstop>muscleLineEdit</tabstop>
  <tabstop>muscleButton</tabstop>
  <tabstop>paddingEditLine</tabstop>
  <tabstop>uniprimerLineEdit</tabstop>
  <tabstop>barcodelenSpinBox</tabstop>
  <tabstop>mindepthSpinBox</tabstop>
  <tabstop>maxdepthSpinBox</tabstop>
  <tabstop>flankinglengthSpinBox</tabstop>
  <tabstop>matchscoreSpinBox</tabstop>
  <tabstop>mismatchscireSpinBox</tabstop>
  <tabstop>gapscoreSpinBox</tabstop>
  <tabstop>maxmismatchSpinBox</tabstop>
  <tabstop>threadSpinBox</tabstop>
  <tabstop>seqfileListWidget</tabstop>
  <tabstop>barcodefileLineEdit</tabstop>
  <tabstop>primerfileLineEdit</tabstop>
  <tabstop>outfolderLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ParameterDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>477</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="filetypeButtonGroup"/>
  <buttongroup name="scoretypeGroup"/>
 </buttongroups>
</ui>
